src/scripts/main.js

```diff
@@ -21,7 +21,23 @@
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const ul = document.createElement('ul');
+
+  for (const key in data) {
+    if (data.hasOwnProperty(key)) {
+      const li = document.createElement('li');
+
+      li.textContent = key;
+
+      if (typeof data[key] === 'object') {
```

Comment:
The check `typeof data[key] === 'object'` is correct for determining if the value is an object. However, this will also be true for `null`, so ensure that `data[key]` is not `null` if that's a possibility in your data structure.